### DISCUSSION TITLE
Tutorial regarding setting up persistent worker

### DISCUSSION THREAD

**Discussion by ghost** [cirruslabs/cirrus-ci-docs#1267] [Q&A]
Hello there! Myself eartinityop.
So I am setting up a persistent worker on my Ubuntu server. I have also installed cirrus-cli and done everything. I am only confused a bit about the .cirrus.yml file. It would really help me if anyone could guide me by giving an example for.cirrus.yml file by which I can use my server as a persistent worker for running CI jobs. I have checked the official tutorial but still I am a bit confused regarding how to place the registration pool token and all in the .yml file. Well my condition is that I don't want an isolated environment. I just want to simply run the worker in the current directory of the server without any isolation. 
Thanks,
Awaiting response

**Comment by fkorotkov** [comment]
Please checkout the following docs:

- https://cirrus-ci.org/guide/persistent-workers/
- https://github.com/cirruslabs/cirrus-cli/blob/master/PERSISTENT-WORKERS.md
